epochs/hippieeg/config/snakebids.yml

```diff
@@ -0,0 +1,124 @@
+bids_dir: '/home/mcesped/projects/ctb-akhanf/cfmm-bids/Khan/epi_iEEG/ieeg'
```

Comment:
You do not need to hard code these paths (bids and output dir), they are populated from the command line inputs when snakebids is run.

Reply:
Sounds good!